pkg/datapath/cells.go

```diff
@@ -60,6 +61,9 @@ var Cell = cell.Module(
 	// The monitor agent, which multicasts cilium and agent events to its subscribers.
 	monitorAgent.Cell,
 
+	// The sysctl reconciler to read and write kernel sysctl parameters.
+	sysctl.Cell,
```

Comment:
Hmm, do we need a fake `Sysctl` (or the real one with fake reconciler)? I guess it'd be great if we didn't need one, but I suspect we might...

Reply:
I think we need one. It should become mandatory as soon as I'll finish the PR plumbing the new `sysctl.Sysctl` into the dependant cells.